redis/redis.go

```diff
@@ -0,0 +1,28 @@
+package redis
+
+import (
+	"context"
+	"log"
+	"os"
+
+	redis "github.com/go-redis/redis/v8"
+)
+
+var Client *redis.Client
+var Nil = redis.Nil
+
+func init() {
+	address := "localhost:6378"
+	if envVal, envFound := os.LookupEnv("REDISCLOUD_URL"); envFound {
+		address = envVal
+	}
+
+	Client = redis.NewClient(&redis.Options{
+		Addr:     address,
+		Password: "",
+		DB:       0,
+	})
```

Comment:
NB there are a bunch of redis options we could tweak later: https://github.com/go-redis/redis/blob/b3e0aa270a144e5452d26eddd75587ef7a9bbd80/options.go#L32-L113